frontend/src/pages/LoginPage/LoginPage.jsx

```diff
@@ -1,36 +1,78 @@
-import React from 'react'
-import { useNavigate } from 'react-router-dom'
-import './loginpage.css'
-import { signInWithEmailAndPassword } from 'firebase/auth'
-import { auth } from '../../lib/firebase'
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './loginpage.css';
+import { signInWithEmailAndPassword, signInWithPopup } from 'firebase/auth';
+import { auth, provider } from '../../lib/firebase';
+import HomePage from "../HomePage/HomePage";
+import Notification from '../../Components/Notification/Notification';
+import { toast } from 'react-toastify';
 
 function LoginPage() {
-  const navigate = useNavigate()
+  const [gmail, setGmail] = useState('');
+  const [email, setEmail] = useState('');
+  const navigate = useNavigate();
 
   const handleLogin = async (e) => {
-    e.preventDefault()
-    const formData = new FormData(e.target)
-    const { email, password } = Object.fromEntries(formData)
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    const { email, password } = Object.fromEntries(formData);
     try {
```

Comment:
Let create custom hook to separate logic and UI 
https://medium.com/@nelson_examiner/building-custom-react-hooks-for-calling-apis-ab82a6b45ff